api/src/chat/controllers/block.controller.ts

```diff
@@ -293,4 +293,29 @@ export class BlockController extends BaseController<
     }
     return result;
   }
+
+  /**
+   * Deletes multiple blocks by their IDs.
+   * @param ids - IDs of blocks to be deleted.
+   * @returns A Promise that resolves to the deletion result.
+   */
+  @CsrfCheck(true)
+  @Delete('')
+  @HttpCode(204)
+  async deleteMany(@Body('ids') ids: string[]): Promise<DeleteResult> {
```

Comment:
Renaming the method from deleteMany to deleteBlocksByIds will be more suitable.

Reply:
Let's keep the naming like this, cause all the names are standardized across all controllers.